src/helpers/logging/logger-options.js

```diff
@@ -5,21 +5,19 @@ import { config } from '~/src/config/index.js'
 /**
  * @satisfies {Options}
  */
-const loggerOptions = {
-  enabled: !config.get('isTest'),
+export const loggerOptions = {
+  enabled: config.get('log.enabled'),
   ignorePaths: ['/health'],
   redact: {
     paths: ['req.headers.authorization', 'req.headers.cookie', 'res.headers'],
     remove: true
   },
-  level: config.get('logLevel'),
-  ...(config.get('isDevelopment')
-    ? { transport: { target: 'pino-pretty' } }
-    : /** @type {Omit<LoggerOptions, 'mixin' | 'transport'>} */ (ecsFormat()))
+  level: config.get('log.level'),
```

Comment:
this is a bit bonkers. Can it be made clearer?

Reply:
possible solution?
```
/**
 * @type {{ecs: Omit<LoggerOptions, "mixin"|"transport">, "pino-pretty": {transport: {target: string}}}}
 */
const formatters = {
  ecs: ecsFormat(),
  'pino-pretty': { transport: { target: 'pino-pretty' } }
}

/**
 * @satisfies {Options}
 */
export const loggerOptions = {
  enabled: config.get('log.enabled'),
  ignorePaths: ['/health'],
  redact: {
    paths: ['req.headers.authorization', 'req.headers.cookie', 'res.headers'],
    remove: true
  },
  level: config.get('log.level'),
  ...formatters[config.get('log.format')]
}
```

Or could have two separate configs, one for ecs one for pino and just have the createLogger decide which to load?

Reply:
The above is a lot easier to read. I'd go with that. Lets see how that goes and then we have "Or could have two separate configs, one for ecs one for pino and just have the createLogger decide which to load?" as an option in the future?